scripts/ci/jobs/gke_version_compatibility_tests.py

```diff
@@ -14,10 +14,11 @@
 # don't use postgres
 os.environ["ROX_POSTGRES_DATASTORE"] = "false"
 
-versions=get_latest_release_versions(4)
+# versions=get_latest_release_versions(4)
+chart_versions=["72.1.0", "71.2.0", "70.0.0"]
```

Comment:
TODO: Probably need to update `get_latest_release_versions` to use `helm search` instead.

Reply:
Are you going to do this in this PR? Otherwise this'll be a chore to periodically update.

Reply:
`helm search repo stackrox-oss -l -o json | jq -r '.[] | select( .name=="stackrox-oss/stackrox-secured-cluster-services" ) | .app_version'` got me:
```
3.72.1
3.72.0
3.71.2
3.71.0
3.70.0
```

Reply:
Are the release versions available on Helm a subset of all available release versions?

Anyways using helm search to get the latest versions is a oneliner now thanks to Gavins command, I think all it should take is plugging that command into the `make_image_tag` function and calling it instead of `subprocess.check_output(["make", "--quiet", "--no-print-directory", "tag"]).decode(encoding="utf-8")`

Reply:
Yep. Just a tiny correction: it should be `.version` instead of `.app_version`.

Reply:
Done